### PR TITLE
Create pull_request_template.md

### LINKED ISSUE #73: Add contributor guidelines
Contributor guidelines should include:
- PR template
- stating what acknowledgement/authorship a contributor is entitled to

### PR DESCRIPTION
**Pull request by hareball90**
Proposed template for a pull request, including guidance for reviewer, slightly edited (removed internal links) from DS QA template.

This partially resolves #73.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+## Checks
+
+This is an itemised checklist for the QA process within UKHSA and represents the bare minimum a QA should be. 
+
+Full instructions on reviewing work can be found at Confluence on the [ONS QA of code guidance](https://best-practice-and-impact.github.io/qa-of-code-guidance/intro.html) page.
+
+**To the reviewer:** Check the boxes once you have completed the checks below.
+
+- [ ] It runs
```
Comment (HWilliams-PHE) [inline .github/pull_request_template.md:14]: passes test

Comment (HWilliams-PHE) [reply .github/pull_request_template.md:14]: tests, coverage, sensible

Comment (twomagpi) [reply .github/pull_request_template.md:14]: These to English

**Review by twomagpi — COMMENTED** [review]
Also needs the broken CI fixed

**Review by twomagpi — DISMISSED** [review]
Single file check list seems sensible

**Comment by hareball90** [comment]
Does /deploy trigger a new workflow?

**Review by twomagpi — APPROVED** [review]
Template meets the internal QA guidelines. May need to be slightly tweaked in longer term for external colaboration.

**Pull request merged**